README.md

```diff
@@ -307,6 +307,11 @@ Table of Contents
   * [tesults.com](https://www.tesults.com) — Test results reporting and test case management. Integrates with popular test frameworks. Open Source software developers, individuals, educators, and small teams getting started can request discounted and free offerings beyond basic free project.
   * [cypress.io](https://www.cypress.io/) - Fast, easy and reliable testing for anything that runs in a browser. Cypress Test Runner is always free and open source with no restrictions and limitations. Cypress Dashboard is free for open source projects for up to 5 users.
   * [percy.io](https://percy.io) - Add visual testing to any web app, static site, style guide, or component library.  10 team members, Demo app and unlimited projects, 5,000 snapshots / month.
+  * [Pingdom](https://www.pingdom.com/) - Monitor your website's availability and performance for free with Pingdom and always be the first to know when your website is down.
+  * [Pagespeed Insights](https://developers.google.com/speed/pagespeed/insights/) - PageSpeed Insights analyzes the content of a web page, then generates suggestions to make that page faster.
+  * [Speedmonitor.io](https://speedmonitor.io) - Track your website speed and performance data every day and track performance over time.
```

Comment:
doesnt appear to be free?

Reply:
It has a free tier like many of the other sites.

Reply:
https://speedmonitor.io/pricing does not mention any free tier